frameworks/Kotlin/hexagon/gradle.properties

```diff
@@ -1,11 +1,11 @@
 description=Hexagon web framework's benchmark
-gradleScripts=https\://raw.githubusercontent.com/hexagonkt/hexagon/0.25.10/gradle
-hexagonVersion=0.25.10
+gradleScripts=https\://raw.githubusercontent.com/hexagonkt/hexagon/1.0.0/gradle
+hexagonVersion=1.0.0
 hikariVersion=3.3.1
-jettyVersion=9.4.15.v20190215
-kotlinCoroutinesVersion=1.1.1
+jettyVersion=9.4.18.v20190429
+kotlinCoroutinesVersion=1.2.1
 kotlinVersion=1.3.31
 logbackVersion=1.2.3
 name=hexagon
-postgresqlVersion=42.2.5
+postgresqlVersion=42.2.5.jre7
```

Comment:
Hi @jaguililla, is there any particular reason for using the Java 7 version of the JDBC driver?
I'm interested in keeping the differences in the used libraries at minimal. Some similar discussion here: https://github.com/TechEmpower/FrameworkBenchmarks/pull/4318